src/app/components/client/PremiumUpsellModal.tsx

```diff
@@ -0,0 +1,173 @@
+import ModalImage from "../client/assets/premium-upsell-modal-icon.svg";
+import styles from "./PremiumUpsellModal.module.scss";
+
+export interface PremiumUpsellModalProps {
+  state: OverlayTriggerState;
+}
+
+function PremiumPricingLabel({ isMonthly }: { isMonthly?: boolean }) {
+  const l10n = useL10n();
+
+  return (
+    <>
+      <small className={styles.pricingInfo}>
+        {l10n.getString(
+          "fix-flow-data-broker-profiles-automatic-remove-save-percent",
+          { percent: 10 }
+        )}
+      </small>
+      <div className={styles.pricingPill}>
+        <div className={styles.pricingLabel}>
+          <b>
+            {l10n.getString(
+              "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-headline"
+            )}
+          </b>
+          <small>
+            {isMonthly
+              ? l10n.getString(
+                  "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-monthly-frequency"
+                )
+              : l10n.getString(
+                  "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-yearly-frequency"
+                )}
+          </small>
+        </div>
+        <b>
+          {l10n.getString(
+            "fix-flow-data-broker-profiles-automatic-remove-features-price",
+            { price: isMonthly ? "X.XX" : "Y.YY" }
+          )}
+        </b>
+      </div>
+    </>
+  );
+}
+
+function PremiumUpsellModalContent() {
+  const l10n = useL10n();
+  const [selectedTab, setSelectedTab] = useState<Key>("yearly");
+
+  const isMonthly = selectedTab === "monthly";
+  const tabsData = [
+    {
+      name: l10n.getString(
+        "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-toggle-yearly"
+      ),
+      key: "yearly",
+      content: <PremiumPricingLabel />,
+    },
+    {
+      name: l10n.getString(
+        "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-toggle-monthly"
+      ),
+      key: "monthly",
+      content: <PremiumPricingLabel isMonthly />,
+    },
+  ];
+
+  const premiumSubscriptionUrl =
+    process.env.FXA_SUBSCRIPTIONS_URL &&
+    process.env.PREMIUM_PRODUCT_ID &&
+    process.env.PREMIUM_PLAN_ID_US &&
+    `${process.env.FXA_SUBSCRIPTIONS_URL}/products/${process.env.PREMIUM_PRODUCT_ID}?plan=${process.env.PREMIUM_PLAN_ID_US}`;
+
+  return (
+    <div className={styles.modalContent}>
+      <div className={styles.productPlans}>
+        <TabList
+          tabs={tabsData}
+          onSelectionChange={(selectedKey) => setSelectedTab(selectedKey)}
+          defaultSelectedKey={selectedTab}
+        />
+      </div>
+      <dl className={styles.list}>
+        <dt>
+          {l10n.getString(
+            "fix-flow-data-broker-profiles-automatic-remove-features-headline"
+          )}
+        </dt>
+        <dd>
+          {l10n.getString(
+            "fix-flow-data-broker-profiles-automatic-remove-features-monthly-scan",
+            { data_broker_count: 190 }
```

Comment:
nit: we've discussed before, but looks like we're now hardcoding this "190" in at least 9 places:

<details>
<summary><kbd>git grep -Pin "\b190\b" -- ":(exclude)locationAutocompleteData.json" ":(exclude)*.svg" | grep -v ":#"</kbd></summary>

```sh
locales-pending/fix.ftl:33:fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph2 = We found these profiles by searching 190 data broker sites for the name, location, and date of birth you provided. These profiles are currently live and available to anyone searching for you.
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx:31:export const ONEREP_DATA_BROKER_COUNT = 190;
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/automatic-remove/page.tsx:19:  const dataBrokerCount = 190;
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/start-free-scan/page.tsx:20:  const dataBrokerCount = 190;
src/app/(proper_react)/redesign/(authenticated)/user/welcome/Onboarding.stories.tsx:21:      dataBrokerCount={190}
src/app/components/client/ExposuresFilterExplainer.tsx:30:          data_broker_sites_total_num: 190,
src/app/components/client/ExposuresFilterExplainer.tsx:61:          data_broker_sites_total_num: 190,
src/app/components/client/PremiumUpsellModal.tsx:107:            { data_broker_count: 190 }
src/app/functions/server/onerep.ts:80:export const ONEREP_DATA_BROKER_COUNT = 190;
```
</summary>

Reply:
Yeah, we have multiple places where we defined the `190`. I put them in to a public env var so we can globally access it in [396df7f](https://github.com/mozilla/blurts-server/pull/3347/commits/396df7f3797d9c49740844cdb59f485e5b61bbfb).